spec/Section 5 -- Validation.md

```diff
@@ -910,7 +910,7 @@ fragment inlineNotExistingType on Dog {
 }
 ```
 
-#### Fragments on Composite Types
+#### Fragments on object, interface or union Types
```

Comment:
See the [style guide on titles](https://github.com/graphql/graphql-spec/blob/main/STYLE_GUIDE.md#title-case):

```suggestion
#### Fragments on Object, Interface or Union Types
```